### PR TITLE
Dg-update-readme

### LINKED ISSUE #22: Update README.md file

### PR DESCRIPTION
**Pull request by dgarcia13-99**
closes #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,121 @@
...
+## Usage
+
+1. Start the Rails server:
+`bin/dev or dev`
+
+2. Open browser and navigate to http://localhost:3000
+
+3. Use one of the following user profiles to interact with data-rich views or create an account and interact with the application using data seeded into your role's interface. 
+
+  NOTE: The credentials provided are for testing purposes only and do not correspond to any real Sephora accounts or information.
```
Comment (hatcheta1) [inline README.md:58]: I like the inclusion of different credentials for each interface

File: README.md [hunk 2]
```diff
@@ -1,3 +1,121 @@
...
+
+9. Admin and Operations teammembers can keep track of tester usage by visiting the `Trashed Testers` view.
+
+## Entity Relationship Diagram 
+
+Visual representation of Tester Tracker's database
+
+![alt text](erd-model.png)
+
+## Troubleshooting
```
Comment (hatcheta1) [inline README.md:87]: nice job including some common issues and how to troubleshoot them

File: Screenshot 2024-12-04 at 9.33.02 AM.png
Comment (hatcheta1) [inline Screenshot 2024-12-04 at 9.33.02 AM.png:1]: I like the inclusion of screenshots to illustrate the features of your app

File: README.md [hunk 3]
```diff
@@ -1,3 +1,121 @@
...
+- [Contributing](#contributing)
+- [Contact](#contact)
+
+## Features
+- Role-specific views for Operations Associates, Admins, Beauty Advisors, Leads, and Managers, each with access to relevant features for their responsibilities.
+- Search for products by name or SKU.
+- Create and manage tester inventory.
+- Trash and approve testers.
+
+## Installation
```
Comment (hatcheta1) [inline README.md:24]: nice clear installation steps

**Review by hatcheta1 — APPROVED** [review]
your readme file is very thorough and seems to cover all the bases for installing your app, using it, and contributing to it

File: README.md [hunk 4]
```diff
@@ -1,3 +1,121 @@
...
+
+### Prerequisites
+- Ruby 3.x
+- Rails 7.x
+- PostgreSQL
+
+1. Clone the repository:
+`git clone https://github.com/dgarcia13-99/tester-tracker-sephora`
+
+2. Navigate to the project directory:
```
Comment (marywebby) [inline README.md]: not sure if this step is needed! unless your project was inside another file when you fork it

File: README.md [hunk 5]
```diff
@@ -1,3 +1,121 @@
...
+
+3. Install the required gems:
+`bundle install`
+
+4. Set up the database:
+`rake sample_data`
+
+5. Start the Rails server:
+`bin/dev`
+
```
Comment (marywebby) [inline README.md:42]: also, are there any configurations to your project? I don't believe you are using any API's at the moment, but maybe put something in here saying like under a new configuration section saying like "no configurations at the moment, if you would like to implement something of that nature,  please refer to the `rails credentials`..." and then link to that

File: README.md [hunk 6]
```diff
@@ -1,3 +1,121 @@
...
+### Unable to log-in
+
+1. Run `rake sample_data` to populate database with seed data.
+
+2. Run `rails console` and query data (e.g. `Employee.all` or `Employee.count`) to ensure it was seeded correctly.
+
+## Contributing
+
+Contributions are welcome! Please follow these guidelines:
+
```
Comment (marywebby) [inline README.md:114]: make sure to include coding conventions, branch naming conventions, and the pull request process for contribution!

**Review by marywebby — APPROVED** [review]
Great job!! Just a few more things to include based off the rubric but apart from that it's great!!

**Pull request merged**